src/Symfony/Component/Routing/Loader/Configurator/ImportConfigurator.php

```diff
@@ -50,6 +51,14 @@ public function __destruct()
         }
         if (!\is_array($prefix)) {
             $this->route->addPrefix($prefix);
+            if (!$trailingSlashOnRoot) {
+                $rootPath = (new Route(trim($prefix).'/'))->getPath();
```

Comment:
The RouteCollection does a little more

https://github.com/symfony/symfony/blob/07a2f6cef445e10ea58506625f7f80fc6f2dd04b/src/Symfony/Component/Routing/RouteCollection.php#L143

You need to do the same thing otherwise it does not work for `/prefix//`. The same applies to the loaders.

Reply:
fixe